profile-doc/profile/scribblings/toplevel.scrbl

```diff
@@ -15,7 +15,8 @@ profiler on the @racket[main] submodule of that file (if it exists), or on the
 module itself (if there is no @racket[main] submodule).
 
 To allow control over its behavior, @exec{raco profile} accepts flags that
-correspond to those of @racket[profile-thunk] below.
+correspond to those of @racket[profile-thunk] below, and additional flags that
+control the order of its output.
```

Comment:
@stamourv I think the new keyword argument is specific to the text renderer (i.e., might make no sense if it were a global option for all renderers). So something like this will work:

```racket
(require (prefix-in text: profile/render-text))
(define (render profile)
  (text:render profile #:order 'self))
(profile ... #:render render)
```